### DISCUSSION TITLE
water stuns should be removed

### DISCUSSION THREAD

**Discussion by deltanedas** [FakeFishGames/Barotrauma#12943] [Ideas and suggestions]
being stunned by water currents is bad:
- if you are trying to man a gun but a little bit of water movement stuns you for 5ms itll take you off guns which is very annoying
- when raiding wrecks that have unflooded rooms you get stunned when opening doors which might give fish inside an opportunity to eat you while you are completely defenseless
- you should be able to flail around while being pushed instead of stunlock

stuns should be for baton and big smacks not a little bit of water

it should just move you, probably exaggerate it a bit more with removal of stuns.
that way you can still shoot but your arms would be pushed around, making it more unpredictable. melee swings would be better in this case

**Comment by SomeRandomNoobKekeke** [comment]
+, tired of being helplessly sucked into husks
also, until recently i didn't realize that most wrecks are not flooded and you can just dock to it and it will be completely dry inside

**Comment by bookslayer10** [comment]
Disagree. It's true that water pushes you around a lot, but that makes perfect sense because water is very heavy. Getting pushed around by hull breaches and swept away in currents is just one of the dangers of working in a partially flooded environment. While it is annoying to be pushed off a periscope, it also heightens the tension and immersion.

If you want to deal with it, items like the safety harness, makeshift armor and exosuit can help mitigate the stun from water. You can also breach doors with a welder, giving you more standoff so you aren't carried with the current. If the gunnery compartment is becoming flooded too often, maybe discourage people from opening the doors in an emergency.

**Reply by deltanedas** [reply]
im all for water pushing you but stunning is silly, why would you not be able to do anything
also when using a periscope your hands would be gripping the handles so no flying off there, just limbs flailing around

**Reply by bookslayer10** [reply]
It's not silly, it's realistic and immersive. Water is very heavy, in real life even just a foot of water crashing into a person could knock them off their feet for a second, and certainly disrupt the task they were doing. Good luck trying to think and act clearly when being tossed like a ragdoll in the current. You'd be ripped off a periscope no matter how hard you are holding on.

If anything it's too lenient with how easy it is to move through water, but these characters have been trained so I guess it makes a bit of sense. Plus there are items to further reduce water pushing players around and stunning, if you don't want to be stunned by the impact then wear a safety harness.

Being pushed around and stunned by water is one of the most common ways that monsters can negatively affect players inside the submarine. Taking it away would remove a significant threat and make encounters much easier. It's dumb to take all of that away for no good reason, even if it is unrealistic.

**Comment by jigardi** [comment]
I disagree with this suggestion as safetyharness specifically addresses water stuns.  However!  I would very much be in favor of an upgrade to the safety harness!  Let's call it >> The Safety Tether << which would allow you to root yourself in place / tie yourself to a submarine, similar to how harpoons work already.

**Reply by NoahRoopull** [reply]
Even an unrelated item that allows you to tether yourself to the sub would be useful (e.g. for the topic of the conversation), but also attaching yourself to the outside (to avoid being left behind while you're repairing the hull, or even for combat in theory).
